app/scripts/controllers/exportImport.js

```diff
@@ -126,6 +126,11 @@ angular.module('openhimConsoleApp')
       var exportData = angular.copy( $scope.selectedExports );
       var textFile = null;
 
+      // OHM-382: warn when exporting key
+      if(exportData !== undefined && exportData.Keystore !== undefined && exportData.Keystore.length > 0) {
+        Alerting.AlertAddMsg('bottom', 'warning', 'the key will be exported and should be protected');
```

Comment:
Also I think make it a top alert, this will involve fixing a bug in the view too. (in views/exportImport.html change line 17 to alert in alerts.top)